program/src/main/System.vala

```diff
@@ -101,35 +99,46 @@ public class System
             var menuItemQuestion = new Gtk.MenuItem.with_label("Question");
             menuItemQuestion.set_submenu(menuQuestion);
             menuBar.append(menuItemQuestion);
+
         }
         
-        
+
         //setup window area UI
         {
             //setup the exam view on left, and tools on right
             var hpaned = new Gtk.Paned(Gtk.Orientation.HORIZONTAL);
             hpaned.set_size_request(800, 400); //minimum size of entire window content
             menuPane.pack2(hpaned, true, false);
             
-            //left side: exam view
+            //left side: exam view AND progressBar?
```

Comment:
Should this question mark be here?

Reply:
Oops, looks like a few of my testing comments went through. Will update all comments in this repo this afternoon - sorry about that! Definitely should've gone through this again before opening the PR